libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -311,7 +308,32 @@ string ABIFunctions::conversionFunction(Type const& _from, Type const& _to)
 			}
 			else if (toCategory == Type::Category::FixedPoint)
 			{
-				solUnimplemented("Not yet implemented - FixedPointType.");
+				FixedPointType const& toFixedPointType = dynamic_cast<FixedPointType const&>(_to);
+				if (fromCategory == Type::Category::Integer)
+				{
+					body =
+						Whiskers("converted := <toFixed>(<cleanInt>(value))")
+						("toFixed", scaleFixedFunction(0, toFixedPointType.fractionalDigits(),
+													   dynamic_cast<IntegerType const &>(_from).isSigned()))
```

Comment:
Please format as follows:
```
x(
    a,
    b,
    c
)
```
I.e. if you want to break a line inside an argument list, put every single argument on its own line, everything that precedes the list and everything that follows the list on its own line. Indentation of the arguments is exactly one tab more than the surrounding lines.